OpenRA.Game/Graphics/SpriteLoader.cs

```diff
@@ -21,6 +22,12 @@ public interface ISpriteLoader
 		bool TryParseSprite(Stream s, out ISpriteFrame[] frames);
 	}
 
+	public interface ISpriteLoader2 : ISpriteLoader
```

Comment:
I'm not sure about this. 
Let me explain. each building in dune2 split into multiple tiles. do sequences support this? this mean that data from .map file should be put into yaml files. So, why we still need icon.icn then? why not to just put everything in newly created .shp files?